runtime/compiler/control/JITClientCompilationThread.cpp

```diff
@@ -1080,6 +1080,7 @@ handleResponse(JITServer::MessageType response, JITServer::ClientStream *client,
          auto recv = client->getRecvData<J9Class*, J9Class*>();
          auto clazz1 = std::get<0>(recv);
          auto clazz2 = std::get<1>(recv);
+         Trc_JITServerVMinlineCastCheckparams(vmThread, clazz1, clazz2);
```

Comment:
I would prefer the name of the tracepoint to reflect the containing code rather than what would be called in the future (i.e. at this point we are not executing the VM helper `inlineCheckCast`).
I wonder whether we should go one step further in the diagnostic and try to deference clazz1 and clazz2 causing a crash on purpose. We could use `TrcEnabled_Trc_JITServerVMinlineCastCheckparams` as a test, issue the trace point and then attempt to print the class name; if the class pointers are bogus, then trying to print the class names will likely result in a crash or some garbage names.
The alternative is to use a debugger and from the coredump to determine whether clazz1 and clazz2 are valid. This would actually be preferred as there is less code clutter.